docs/src/components/Layout/FrameworkChooser.tsx

```diff
@@ -14,6 +14,17 @@ interface FrameworkLinkProps extends FrameworkChooserProps {
 
 const platformPath = '[platform]';
 
+const getFrameworkDisplayName = (framework: string) => {
+  // if framework is more than one word, capitalize all words
+  if (framework?.includes(' ')) {
+    return framework
+      .split(' ')
+      .map((item) => capitalize(item))
+      .join(' ');
+  }
+  return capitalize(framework);
+};
+
```

Comment:
Should we memoize this function via useCallback?

Reply:
Since `getFrameworkDisplayName ` is declared outside the scope of `FrameworkChooser` its reference does not change so pretty sure we do not need to wrap in a `useCallback`, and because it always returns a string (and therefore will not trigger re-renders) _think_ we are okay with not memoizing the output with `useMemo`. Thoughts?

Reply:
@ioanabrooks  We could also just have use an object to look up the current framework name:
```
const FRAME_WORK_DISPLAY_NAMES = {
  react: 'React',
  'react native': 'React Native',
  flutter: 'Flutter',
  // ...other values
}
```

Not sure if we really need the complexity of the function I added when we know exactly what the `framework` values are